api/progress.go

```diff
@@ -67,6 +67,9 @@ func configProgressRouter(router *gin.Engine, daoWrapper dao.DaoWrapper) {
 	go progressBuff.run()
 	router.POST("/api/progressReport", routes.saveProgress)
 	router.POST("/api/watched", routes.markWatched)
+
+	router.POST("/api/seekReport/:streamID", routes.reportSeek)
```

Comment:
Could you add some tests for those two routes? The tests will be pretty similar to the ones in `search_test.go`, so you might want to have a look at them.